beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/Eth1DataCache.java

```diff
@@ -124,6 +126,10 @@ public Eth1Data getEth1Vote(BeaconState state) {
     return vote.orElse(defaultVote);
   }
 
+  public List<Eth1Data> getAllEth1Blocks() {
+    return new ArrayList<>(this.eth1ChainCache.values());
```

Comment:
I think you can return `Collection<Eth1Data>` here. And keep returning it up to the handler.  SerializableTypeDefinition should handle it

Reply:
Done